catalog/admin/catalog-admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -154,7 +154,8 @@ public boolean sourceStatus(String servicePID) {
               try {
                 return source.isAvailable();
               } catch (Exception e) {
-                LOGGER.debug("Couldn't get availability on source {}: {}", servicePID, e);
+                LOGGER.debug(
```

Comment:
why do we add `String.format`?  Can't we just drop the second set of braces?

Reply:
I don't think you can mix message arguments and a throwable. 

```java
/**
 * Log a message at the DEBUG level according to the specified format
 * and arguments.
 * <p/>
 * <p>This form avoids superfluous object creation when the logger
 * is disabled for the DEBUG level. </p>
 *
 * @param format the format string
 * @param arg1   the first argument
 * @param arg2   the second argument
 */
public void debug(String format, Object arg1, Object arg2);
```
https://www.javadoc.io/static/org.slf4j/slf4j-api/1.7.1/org/slf4j/Logger.html#debug(java.lang.String,%20java.lang.Object,%20java.lang.Object)

```java
/**
 * Log an exception (throwable) at the DEBUG level with an
 * accompanying message.
 *
 * @param msg the message accompanying the exception
 * @param t   the exception (throwable) to log
 */
public void debug(String msg, Throwable t);
```
https://www.javadoc.io/static/org.slf4j/slf4j-api/1.7.1/org/slf4j/Logger.html#debug(java.lang.String,%20java.lang.Throwable)

Reply:
`LOGGER.debug("Couldn't get availability on source {}", servicePID, e);` should work; we do it all the time

Reply:
You're right. I just confirmed that it gets logged the same either way, even with more than one message argument :+1: I'd like to keep it using String.format for these ones that I fixed so that it aligns with the javadoc.

Reply:
We do add extra processing cost by wrapping a String.format to these log statements

Reply:
Fixed :+1: